src/main/java/org/gridsuite/filter/utils/expertfilter/FieldType.java

```diff
@@ -115,7 +115,6 @@ public enum FieldType {
     SUBSTATION_PROPERTIES,
     SUBSTATION_PROPERTIES_1,
     SUBSTATION_PROPERTIES_2,
```

Comment:
```suggestion
    SUBSTATION_PROPERTIES_2,
    SUBSTATION_PROPERTIES_3, // this FieldType value is obsolete but kept here to avoid crashes in case of remaining filters using it
```